.github/workflows/CI.yml

```diff
@@ -18,13 +18,31 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
 
+    - uses: fortran-lang/setup-fpm@v4
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+
     - name: Install Dependencies MacOS
       if: contains(matrix.os, 'mac')
       run: |
         brew install gcc@${GCC_V}
         sudo ln -s $(which gfortran-${GCC_V}) $(dirname $(which gfortran-${GCC_V}))/gfortran
 
-    - name: Build and Test
+    - name: Install on Ubuntu
+      if: contains(matrix.os, 'ubuntu')
+      run: |
+        sudo apt update
+        sudo apt install -y build-essential gfortran-14 gcc-14 g++-14
+
+    - name: Build and Test MacOS
+      if: contains(matrix.os, 'mac')
       run: |
         export PATH="${HOME}/.local/bin:$PATH"
         ./setup.sh
+
+    - name: Build and Test Ubuntu
+      if: contains(matrix.os, 'ubuntu')
+      run: |
+        export FPM_FC=gfortran-14
```

Comment:
```suggestion
        fpm --version
        export FPM_FC=gfortran-14
```

Reply:
@ktras given that the block at line 21 doesn't explicitly make clear to the reader which version of `fpm` is being installed, it might be nice to show the version number here so that we can see it when reading the test transcripts.